apps/opik-backend/src/test/java/com/comet/opik/api/resources/utils/StatsUtils.java

```diff
@@ -85,6 +90,20 @@ public static List<ProjectStatItem<?>> getProjectSpanStatItems(List<Span> expect
                 Span::tags,
                 Span::startTime,
                 Span::endTime,
+                span -> {
+                    if (span.model() != null) {
```

Comment:
Before we added `model` field recently, this was included in `metadata`.
So for getting model would be better to use:
```
String model = StringUtils.isNotBlank(span.model())
                ? span.model()
                : Optional.ofNullable(span.metadata())
                        .map(metadata -> metadata.get("model"))
                        .map(JsonNode::asText).orElse("");
```